05-html-css/06-css-selectors.md

```diff
@@ -39,67 +41,90 @@ Attribute selectors can get more specific, read [this article](https://css-trick
 ### Pseudo-classes
 A CSS pseudo-class is a keyword added to selectors that specifies a special state of the element to be selected.
 
-[Here](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes) is an index of pseudo-classes.
+[MDN maintains an index of pseudo-classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes).
+
+A popular one is `:hover`. Test it out in [Codepen](http://codepen.io) to see how it works!
 
-A popular one is :hover. Test it out in [Codepen](http://codepen.io) to see how it works!
 ```css
 h1:hover { color: teal; }
 ```
+
 ### Pseudo-elements
-Pseudo-elements allow you to style certain parts of a document. A few common pseudo-elements are :after, :before and :first-letter. [Here](https://developer.mozilla.org/en-US/docs/Web/CSS/pseudo-elements) is an index of Pseudo-elements
+Pseudo-elements allow you to style certain parts of a document. A few common pseudo-elements are `::after`, `::before` and `::first-letter`. [MDN also maintains an index of pseudo-elements](https://developer.mozilla.org/en-US/docs/Web/CSS/pseudo-elements).
 
 - Note: Pseudo-elements use two colons, pseudo-classes use only one colon.
 
 Try this out in [Codepen](http://codepen.io)
+
 ```css
 h1::selection {
   color: yellow;
   background-color: black;
 }
 ```
 
+**Question:** What is the difference between a pseudo-element and a pseudo-class? Why are they different things at all?
+
+## Selectors Based on Relationships
+Remember how we emphasized elements relationship with the sectioning elements lesson? Knowing relationships between elements gives us even more ways of selecting elements!
+
+Here are common selectors based on relationships
+
+| Selector        | Selects        
```

Comment:
I don't know that it's really needed here - other than `A E`, I would consider these as "good to know it exists but you'll never actually use it". Especially since any examples will need both HTML and CSS, I think that anything we add will be a bunch of extra clutter, and not add a lot to student understanding.